apps/docs/src/guide/transactions/index.md

```diff
@@ -0,0 +1,14 @@
+# Transactions
+
+A transaction is a way of interacting with Fuel Network's blockchain and can include actions like transferring assets, deploying contracts and minting tokens.
+
+Within the Fuel Network, we have the following transaction types:
+
+- Script
+- Create
+- Mint
+
+The SDK provides class helpers for handling script and create transactions: `ScriptTransactionRequest` and `CreateTransactionRequest`, respectively.
+Mint transactions can only be created by the block producer and do not have any use outside of block creation. Therefore, the SDK only provides the ability to decode them.
```

Comment:
Feel like this paragraph should be:

1. A _note_ rather than a main paragraph
2. Below the paragraph which is currently the last on this page

Reply:
I feel this way because this paragraph contains informationa about more advanced concepts like customt script transactions etc. We should tell users about the 'straightforward' transactions first, which is what the last paragraph does.

Reply:
Made some changes [here](https://github.com/FuelLabs/fuels-ts/pull/1961/commits/7b20becfd6431fc1f8ff726d812551ec6b66c2ff) based on these comments.